recipes/paraprobe/meta.yaml

```diff
@@ -0,0 +1,93 @@
+{% set name = "paraprobe" %}
+{% set version = "v0.3.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://gitlab.com/{{ name }}/{{ name }}-toolbox/-/archive/{{ version }}/{{ name }}-toolbox-{{ version }}.tar.gz
+    sha256: ded5193f54f6aa63fea3092f0cf8a680d2ad96501936e830ef1ba7f298595577
+    folder: paraprobe-toolbox
+    patches:
+      - paraprobe.patch
+  - fn: v0.4.6.tar.gz
+    url: https://github.com/chr1shr/voro/archive/refs/tags/v0.4.6.tar.gz
+    folder: voro++
+    sha256: e1eb9c1f5a393604f2ca56e935b7e850644f7a3a18f5a85a5b2617f51fc70343
+
+build:
+  number: 0
+  skip: true  # [not linux]
+  #skip: True  # [osx]
+  #skip: True  # [py<38] 
+  {% if mpi != "nompi" %}
+  {% set mpi_prefix = "mpi_" + mpi %}
+  {% else %}
+  {% set mpi_prefix = "nompi" %}
+  {% endif %}
```

Comment:
You probably want the mpi variant in the build string. See our docs for how to do this.

Reply:
Thanks, added the build string following the documentation.